libafl/src/bolts/os/windows_exceptions.rs

```diff
@@ -315,8 +318,12 @@ unsafe fn internal_handle_exception(
     }
 }
 
-/// Internal function that is being called whenever an exception arrives (stdcall).
-unsafe extern "system" fn handle_exception(exception_pointers: *mut EXCEPTION_POINTERS) -> c_long {
+/// Function that is being called whenever an exception arrives (stdcall).
+/// # Safety
+/// This function is unsafe because it is called by the OS
+pub unsafe extern "system" fn handle_exception(
```

Comment:
is it ok to be public? @tokatoka we could also minimize the damage by using `pub(crate)` I think

Reply:
I think it's ok.